receiver/kubeletstatsreceiver/README.md

```diff
@@ -16,6 +16,10 @@
 The Kubelet Stats Receiver pulls node, pod, container, and volume metrics from the API server on a kubelet
 and sends it down the metric pipeline for further processing.
 
+> [!WARNING]
+> The `receiver.kubeletstats.enableCPUUsageMetrics` feature gate was moved to stage `Beta` what results in a breaking change in metrics names.
```

Comment:
I think it should mention what version enables the feature gate by default.

Reply:
> 1. The feature gate has nothing to do with the actual receiver configuration.

Not with the configuration but with effects for end users.

> 2. The concrete version when the actual change happens in described at the bottom of the README, I guess this might be enough. WDYT?

Ok so you already added it. I didn't see it before. Thanks!

Reply:
My comment about config validation is this one https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39488#discussion_r2055961896

I think there it makes sense to do it as part of the config validation because a user can validate their configuration and not wait until the collector is executed. But is a nit.

Reply:
> Yeah but I think this should not be part of config validation...

I kind of agree: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39488#discussion_r2055961896

But having it here in the docs, as you did, can help users to "discover" faster something changed.